packages/elements/list/src/transforms/toggleList.ts

```diff
@@ -1,46 +1,139 @@
 import {
   ELEMENT_DEFAULT,
+  findNode,
   getNodes,
+  isCollapsed,
   setNodes,
-  someNode,
   wrapNodes,
 } from '@udecode/plate-common';
 import { getPlatePluginType, SPEditor, TElement } from '@udecode/plate-core';
-import { Editor } from 'slate';
+import { Editor, Node, NodeEntry, Path, Range } from 'slate';
 import { ELEMENT_LI, ELEMENT_LIC } from '../defaults';
+import { getListItemEntry, getListTypes } from '../queries';
 import { unwrapList } from './unwrapList';
 
 export const toggleList = (editor: SPEditor, { type }: { type: string }) => {
-  if (!editor.selection) return;
+  if (!editor.selection) {
```

Comment:
Moving this check inside of Editor.withoutNormalizing has 2 benefits:

1. all as Range casts can be removed
2. brackets can be removed from toggleList, decreasing indentation